app/templates/base.html

```diff
@@ -7,7 +7,8 @@
       {% endif %}
     </head>
     <body>
-        <div>ToDo: <a href="/index">Home</a></div>
+        <div>ToDo: <a href="/home">Home</a> <a href='/login'>Login</a>  <a href="/logout">Logout</a></div>
```

Comment:
If you are going to use React, then do not spend a lot of time on jinja templating..
Make sure that the APIs are working first.